nservicebus/upgrades/all-versions.include.md

```diff
@@ -319,14 +319,6 @@ No versions released.
 | [1.1.x](https://www.nuget.org/packages/NServiceBus.Transport.Msmq/1.1.1) | 2020-06-24     | -                 | -                                 |
 | [~~1.0.x~~](https://www.nuget.org/packages/NServiceBus.Transport.Msmq/1.0.2) | ~~2018-05-29~~ | ~~2020-09-24~~    | ~~Superseded by 1.1.x~~           |
 
-#### [Particular.TimeoutMigration](/nuget/Particular.TimeoutMigration)
```

Comment:
You could take changes to these auto-generated files out and let the overnight InternalAutomation process take care of them.